apps/www/src/queries/index.js

```diff
@@ -1,14 +1,7 @@
 import { hydra } from '@tpluscode/rdf-ns-builders'
-import ParsingClient from 'sparql-http-client/ParsingClient.js'
 
-export const endpoint = 'https://ld.integ.stadt-zuerich.ch/query'
-
-export const client = new ParsingClient({
```

Comment:
I'm trying to understand the client in the state :)

Wouldn't a ParsingClient singleton that instantiates using env.SOME_ENDPOINT do the work?, or might it change at runtime?

Reply:
It quite the opposite. `env.SOME_ENDPOINT` does not exist in runtime because `env` is a node concept.

The runtime is the browser and unless you inject the `process.env` during webpack build, there will be no value for this

Reply:
I meant ``process.env` , sorry :)